helm-chart/renku/templates/secrets.yaml

```
@@ -1,4 +1,11 @@
 ---
+{{- $data_service_kc_client_secret := randAlphaNum 64 | b64enc | quote }}
+
+{{- $renku_secret := lookup "v1" "Secret" .Release.Namespace (include "renku.fullname" .) }}
+{{- if $renku_secret }}
```

trying to fix new deployments:
```suggestion
{{- if and ($renku_secret) ($renku_secret.data) }}
```